public/stylesheets/style.css

```diff
@@ -1,595 +1,1334 @@
 body {
```

Comment:
@thinkking and @nick22891 I noticed that on nearly every commit that you guys do, your editors make modification to the amount of tab spaces used.  I suggest we standardize to 4 tab spaces, so that we reduce the number of unnecessary code changes done at the repository.  

The changing of tab spacing gets reflected as work done and results in an inaccurate measure of a developer's productivity.